server/Helpers/signIn.js

```diff
@@ -13,7 +20,7 @@ export const setAccesTokenCookie = (res, userId) => {
   const accesToken = createAccestToken(userId);
 
   res.cookie("accesToken", accesToken, {
-    httpOnly: true,
+    ...accesTokenOption,
     // todo: add `secure: true` to only send token in https
     // secure: true,
```

Comment:
Delete this comment, it's already in `accesTokenOption`, we only need it one time